cms/admin/placeholderadmin.py

```diff
@@ -695,8 +695,15 @@ def delete_plugin(self, request, plugin_id):
             "opts": opts,
             "app_label": app_label,
         }
-        return TemplateResponse(request, "admin/cms/page/plugin/delete_confirmation.html", context,
-                                current_app=self.admin_site.name)
+        request.current_app = self.admin_site.name
+        if DJANGO_1_7:
+            return TemplateResponse(
+                request, "admin/cms/page/plugin/delete_confirmation.html", context, current_app=self.admin_site.name
+            )
+        else:
+            return TemplateResponse(
```

Comment:
Depending on the outcome of the flake8 max line length discusison, this is as example of where I'd use a single longer line (103 here).